uniffi/src/scaffolding.rs

```diff
@@ -77,7 +77,7 @@ mod filters {
         // By explicitly naming the type here, we help the rust compiler to type-check the user-provided
         // implementations of the functions that we're wrapping (and also to type-check our generated code).
         Ok(format!(
-            "<{} as uniffi::support::ViaFfi>::into_ffi_value({})",
+            "<{} as uniffi::support::ViaFfi>::into_ffi_value(&{})",
```

Comment:
I'm wondering whether it's best to have `into_ffi_value` borrow `self`, or make `Lowerable::lower_into` own `self`. Asking because the `Into::into` in the standard library takes ownership, and I'm not sure if that does some stack memory magic 😆. Probably doesn't make a difference, but wondering what you think!